pkg/apiutil/client.go

```diff
@@ -68,3 +69,20 @@ func (c *Client) GetLeader() (*pdpb.Leader, error) {
 	}
 	return leader, nil
 }
+
+// GetClusterID returns the Cluster ID.
+func (c *Client) GetClusterID() (uint64, error) {
+	clusterURL := c.url + "/cluster"
```

Comment:
`/cluster` does not need bootstrap. It's tso request that needs bootstrap, can we relax restrictions for tso?